exporter_test.go

```diff
@@ -724,13 +724,11 @@ func getFloat64(metrics []*dto.MetricFamily, name string, labels prometheus.Labe
 	}
 
 	var metric *dto.Metric
-	sortedLabelNames := getSortedLabelNames(labels)
-	labelsHash := hashNameAndLabels(name, sortedLabelNames, labels)
```

Comment:
`getSortedLabelNames` and `hashNameAndLabels` no longer exist.  Printing the labels into a string is slow but produces a sorted representation with less complexity.